### PR TITLE
{Feedback} Standardize GitHub bug report and az feedback

### PR DESCRIPTION
**Pull request by bebound**
**Description**

GitHub bug report and az feedback are primary feedback channels. But their fields are different.
This PR makes bug report template identical with az feedback.

The added `errors` and `Command Name` sections help us to investigate the error.

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/feedback/custom.py [hunk 1]
```diff
@@ -70,6 +70,9 @@
 ```
 Paste here the error message you have received. Make sure to remove all sensitive information,
 such as user name, password, credential, subscription ID, etc.
+
+The command failed with an unexpected error. Here is the traceback:
+{The rest of the error message}
```
Comment (bebound) [inline src/azure-cli/azure/cli/command_modules/feedback/custom.py:75]: The error message starts with `The command failed with an unexpected error. Here is the traceback`. Hope it helps the user to copy the full error massage.

**Comment by yonzhan** [comment]
GitHub bug report improvement

**Review by wangzelin007 — APPROVED** [review]

**Comment by jsntcy** [comment]
Is it possible that we only have one bug template and `az feedback` just embeds the content of the bug template instead of change multiple places.

**Comment by bebound** [comment]
Close as https://github.com/Azure/azure-cli/pull/26463 is merged

**Pull request closed without merging**